ui-modules/blueprint-composer/app/components/quick-fix/quick-fix.js

```diff
@@ -353,7 +398,7 @@ export function getQuickFixProposer(type) {
 export function getQuickFixHintsForIssue(issue, entity) {
     if (issue.group === 'config') {
         let hints = (entity.miscData.get('ui-composer-hints') || {})['config-quick-fixes'] || [];
-        hints = hints.filter(h => h.key === issue.ref);
+        hints = hints.filter(h => new RegExp(h.key).test(issue.ref));
```

Comment:
Puzzling myself with this RegExp. It supposed to take regex expression as a parameter, however, `h.key` is a plain string if I understand that correctly. Does it work? What do we filter here?

Reply:
@algairim `RegExp` can take a regex or a string as regex, this is perfectly valid. The idea is to filter for the specific config key but in the case of implicit use of inherited config key, we want to do the check for ALL config key (see: https://github.com/apache/brooklyn-ui/pull/241/files#diff-b9b9776bc3d54d479c21091094606ac270b807327ee31a93317a3598964e2749R39)

This is merly to be backward compatible with what was there previously. The quick fix API is super obscure and I plan to surface this kind of thing much more when refactoring it

Reply:
I see, thanks for clarifying.